data/elan-42ab.tablet

```diff
@@ -8,13 +7,12 @@
 
 [Device]
 Name=Dell Inc. Inspiron 14 7445 2-in-1
-ModelName=
-DeviceMatch=i2c|04f3|42ab
+ModelName=ELAN3233:00 04F3:42AB
+DeviceMatch=i2c|04f3|42ab;i2c|04f3|42ab Stylus
 Class=ISDV4
 Width=12
 Height=7
 IntegratedIn=Display;System
-Styli=@isdv4-aes;
```

Comment:
so it doesn't identify any stylus tools? If so it should use the `@generic` group

Reply:
I found out @isdv4-aes only lists styli with one button when using Dell PN7522w which was 2 buttons and no physical eraser. I tried @generic-no-eraser but it seems missing tilt and pressure. @generic seems a logical choice.